lib/manager/travis/package.js

```diff
@@ -2,20 +2,60 @@ const is = require('@sindresorhus/is');
 const { isEqual } = require('lodash');
 const { getPkgReleases } = require('../../datasource');
 const { isVersion, maxSatisfyingVersion } = require('../../versioning/semver');
+const nodeJsSchedule = require('./data/node-js-schedule.json');
 
 module.exports = {
   getPackageUpdates,
 };
 
-// Start version numbers as integers for correct sorting
-const policies = {
-  all: [8, 10, 11, 12],
-  lts: [8, 10],
-  active: [10, 12],
-  lts_active: [10],
-  lts_latest: [10],
-  current: [12],
-};
+let policies;
+
+function generatePolicies() {
+  policies = {
+    all: [],
+    lts: [],
+    active: [],
+    lts_active: [],
+    lts_latest: [],
+    current: [],
+  };
+
+  const now = new Date();
+
+  for (const [vRelease, data] of Object.entries(nodeJsSchedule)) {
+    const isAlive = new Date(data.start) < now && new Date(data.end) > now;
+    if (isAlive) {
+      const release = parseInt(vRelease.replace(/^v/, ''), 10);
+      policies.all.push(release);
+      const isMaintenance = new Date(data.maintenance) < now;
+      if (!isMaintenance) policies.active.push(release);
+      const isLts = data.lts && new Date(data.lts) < now;
+      if (isLts) {
+        policies.lts.push(release);
+        if (!isMaintenance) policies.lts_active.push(release);
+      }
+    }
+  }
+  policies.current.push(policies.active[policies.active.length - 1]);
```

Comment:
Or sort here?

Reply:
Node.js are putting them in the correct order in schema.json for now, I think I'll leave it